### ISSUE TITLE
Illegal folder name in Create: Interactive JAR file.

### ISSUE THREAD

**Issue by Tjmaxmillion** [ValkyrienSkies/Create-Interactive-Issues#69]
Windows has some reserved folder/file names, and when folders or files are named these things it can cause issues.

In this case `Create: Interactive` has a folder called `aux` located at `org\valkyrienskies\create_interactive\aux` in the jar.
This specific issue popped up with the [HeliosLauncher](https://github.com/dscalzi/HeliosLauncher), The launcher unarchives mods JAR files and stores them in a folder called "modstore".

This makes a folder called `aux` in the folder for `Create: Interactive` preventing the deletion of the root directory of the launcher.

## TL;DR 
Don't use Illegal file/folder names listed below.
Cause if the mod is unarchived on a windows machine for any reason it prevents the folder and any parent folder from being deleted by normal means.

[Image Source](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file)
![image](https://github.com/ValkyrienSkies/Create-Interactive-Issues/assets/76654305/4e1cd5e7-0a66-4c96-8901-b25f021a6c4c)

**Comment by blockninja124** [comment]
I can't find this folder name in source. Can you tell me what mod loader and minecraft version your using? If I still can't find it, its possible its a Kotlin bug (compiling stuff to an aux folder when it shouldn't) or this was already fixed earlier and this issue just wasn't closed.
